api/Startup.cs

```diff
@@ -28,8 +24,32 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddControllers();
 
+            services.AddDbContext<HotelsDatabaseContext>(
+                options => options.UseSqlServer(Configuration.GetConnectionString("HotelsDb"))
+            );
+
             var authOptionsConfiguration = Configuration.GetSection("Auth");
             services.Configure<AuthOptions>(authOptionsConfiguration);
+            var authOptions = authOptionsConfiguration.Get<AuthOptions>();
+
+            services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
+                .AddJwtBearer(options =>
+                {
+                    options.RequireHttpsMetadata = false; // should true for real server
+                    options.TokenValidationParameters = new TokenValidationParameters
+                    {
+                        ValidateIssuer = true,
+                        ValidIssuer = authOptions.Issuer,
+
+                        ValidateAudience = true,
+                        ValidAudience = authOptions.Audience,
+
+                        ValidateLifetime = true,
+
+                        IssuerSigningKey = authOptions.GetSymmetricSecurityKey(),
+                        ValidateIssuerSigningKey = true,
+                    };
+                });
```

Comment:
In my opinion code style can be improved. What do you think @andrei-kruhliakou-itechart?
```suggestion
            services
                .AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
                .AddJwtBearer(
                    options =>
                    {
                        options.RequireHttpsMetadata = false; // should true for real server
                        options.TokenValidationParameters = new TokenValidationParameters
                        {
                            ValidateIssuer = true,
                            ValidIssuer = authOptions.Issuer,

                            ValidateAudience = true,
                            ValidAudience = authOptions.Audience,

                            ValidateLifetime = true,

                            IssuerSigningKey = authOptions.GetSymmetricSecurityKey(),
                            ValidateIssuerSigningKey = true,
                        };
                    }
                );
```

Reply:
There are no limits to code style improvements. :)
Personally I like @bagatka style more, but initial style by @TatsianaMaltseva is also fine and used by many teams.
The most important thing here - you should choose one and follow it for the particular project.

Reply:
Well I like this code style pretty much too))